### ISSUE TITLE
Permission error with successfull directory creation

### ISSUE THREAD

**Issue by michardy** [michardy/note-digest#6]
This is probably an upstream error.  

After successful creation of the target directory this is thrown:
```
thread 'main' panicked at 'Output Generation: error creating root index: Error { repr: Os { code: 13, message: "Permission denied" } }', /checkout/src/libcore/result.rs:916:4
```

**Comment by michardy** [comment]
Never mind.  The error messages were too similar.  The error was in creating a file in the directory using the wrong path

**Comment by michardy** [comment]
Fixed ages ago by: 23546aec8d06e3a36e3c4a009bd29e746b788a67
